src/pages/index.html

```diff
@@ -0,0 +1,107 @@
+          fill="#152B79" />
+        <path
+          d="M54.4915 6.792C54.9395 6.168 55.5395 5.696 56.2915 5.376C57.0595 5.056 57.9395 4.896 58.9315 4.896V8.352C58.5155 8.32 58.2355 8.304 58.0915 8.304C57.0195 8.304 56.1795 8.608 55.5715 9.216C54.9635 9.808 54.6595 10.704 54.6595 11.904V18H50.9155V5.088H54.4915V6.792ZM67.1845 18.192C65.8245 18.192 64.6005 17.912 63.5125 17.352C62.4405 16.776 61.6005 15.984 60.9925 14.976C60.3845 13.968 60.0805 12.824 60.0805 11.544C60.0805 10.264 60.3845 9.12 60.9925 8.112C61.6005 7.104 62.4405 6.32 63.5125 5.76C64.6005 5.184 65.8245 4.896 67.1845 4.896C68.5445 4.896 69.7605 5.184 70.8325 5.76C71.9045 6.32 72.7445 7.104 73.3525 8.112C73.9605 9.12 74.2645 10.264 74.2645 11.544C74.2645 12.824 73.9605 13.968 73.3525 14.976C72.7445 15.984 71.9045 16.776 70.8325 17.352C69.7605 17.912 68.5445 18.192 67.1845 18.192ZM67.1845 15.12C68.1445 15.12 68.9285 14.8 69.5365 14.16C70.1605 13.504 70.4725 12.632 70.4725 11.544C70.4725 10.456 70.1605 9.592 69.5365 8.952C68.9285 8.296 68.1445 7.968 67.1845 7.968C66.2245 7.968 65.4325 8.296 64.8085 8.952C64.1845 9.592 63.8725 10.456 63.8725 11.544C63.8725 12.632 64.1845 13.504 64.8085 14.16C65.4325 14.8 66.2245 15.12 67.1845 15.12ZM89.2315 5.088L83.3995 18.792C82.8075 20.28 82.0715 21.328 81.1915 21.936C80.3275 22.544 79.2795 22.848 78.0475 22.848C77.3755 22.848 76.7115 22.744 76.0555 22.536C75.3995 22.328 74.8635 22.04 74.4475 21.672L75.8155 19.008C76.1035 19.264 76.4315 19.464 76.7995 19.608C77.1835 19.752 77.5595 19.824 77.9275 19.824C78.4395 19.824 78.8555 19.696 79.1755 19.44C79.4955 19.2 79.7835 18.792 80.0395 18.216L80.0875 18.096L74.4955 5.088H78.3595L81.9835 13.848L85.6315 5.088H89.2315ZM103.56 0.191999V18H99.9837V16.512C99.0557 17.632 97.7117 18.192 95.9517 18.192C94.7357 18.192 93.6317 17.92 92.6397 17.376C91.6637 16.832 90.8957 16.056 90.3357 15.048C89.7757 14.04 89.4957 12.872 89.4957 11.544C89.4957 10.216 89.7757 9.048 90.3357 8.04C90.8957 7.032 91.6637 6.256 92.6397 5.712C93.6317 5.168 94.7357 4.896 95.9517 4.896C97.5997 4.896 98.8877 5.416 99.8157 6.456V0.191999H103.56ZM96.5997 15.12C97.5437 15.12 98.3277 14.8 98.9517 14.16C99.5757 13.504 99.8877 12.632 99.8877 11.544C99.8877 10.456 99.5757 9.592 98.9517 8.952C98.3277 8.296 97.5437 7.968 96.5997 7.968C95.6397 7.968 94.8477 8.296 94.2237 8.952C93.5997 9.592 93.2877 10.456 93.2877 11.544C93.2877 12.632 93.5997 13.504 94.2237 14.16C94.8477 14.8 95.6397 15.12 96.5997 15.12Z"
+          fill="#3250BD" />
+      </svg>
+    </h2>
```

Comment:
Where's the text for heading? That's not a heading, it's just a logo. Here's [helpful article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).